### PR TITLE
fix: make mentions with nicknames work the same as mobile

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ui/shared/status/StatusChatInput.qml [hunk 1]
```diff
@@ -439,12 +439,16 @@ Rectangle {
               chatsModel.plainText(Emoji.deparse(messageInputField.text)) :
               chatsModel.plainText(messageInputField.text);
 
-            let aliasName = item[suggestionsBox.property.split(",").map(p => p.trim()).find(p => !!item[p])]
+
+
+            var properties = "ensName, alias"; // Ignore localNickname
+
+
```
Comment (jrainville) [inline ui/shared/status/StatusChatInput.qml:446]: That's a lot of spaces.

**Review by jrainville — APPROVED** [review]
All good, but man, does that line need that much social distancing? Does it have leprosy??

File: ui/shared/status/StatusChatInput.qml (hunk 1 above)
Comment (richard-ramos) [reply ui/shared/status/StatusChatInput.qml]: Code was feeling claustrophobic

**Review by jrainville — APPROVED** [review]
> asdad

Nice commit name

**Pull request merged**